test/integration/file-utils.spec.js

```diff
@@ -75,11 +75,11 @@ describe('file utils', function() {
       var dirLookup = function() {
         return utils.lookupFiles(tmpDir, undefined, false);
       };
-      expect(
-        dirLookup,
-        'to throw',
-        'extensions parameter required when filepath is a directory'
-      );
+      expect(dirLookup, 'to throw', {
+        message: 'Parameter required when filepath is a directory',
```

Comment:
Should we remove the `message`? The _point_ was to interrogate the `code` so spelling, etc. could safely be ignored.

Reply:
Thanks for the "how to" assertion example.

Reply:
For this test its important to check the `argument` property here too so can still use the default
```
expect(dirLookup, 'to throw', {
  code: 'ERR_MOCHA_MISSING_ARG_TYPE',
  argument: 'extensions'
});
```

That new assertion type looks useful though.